src/drivers/distance_sensor/tfmini/tfmini.cpp

```diff
@@ -730,7 +715,8 @@ start(const char *port, uint8_t rotation)
 	int fd;
 
 	if (g_dev != nullptr) {
-		errx(1, "already started");
+		PX4_ERR("already started");
+		exit(1);
```

Comment:
Can you drop the exits while you're at it? This is otherwise close fo functional on a raspberry pi.